app/resources/events/event.rb

```diff
@@ -2,10 +2,10 @@
 # Models events which modify resources
 
 class Event < Valkyrie::Resource
-  attribute :type
-  attribute :status
-  attribute :resource_id
-  attribute :child_property
-  attribute :child_id
-  attribute :message
+  attribute :type, Valkyrie::Types::String
```

Comment:
These break the test suite (which expect `Arrays` rather than `Strings`), please see https://circleci.com/gh/pulibrary/figgy/15899#tests/containers/1

Reply:
Yup, taken care of.